src/github.com/ebay/beam/diskview/keys/keys_test.go

```diff
@@ -210,15 +210,15 @@ func Test_FactKeyBytes(t *testing.T) {
 	}
```

Comment:
There might be code or comments to update in `diskview/lookup.go`'s `lookupSP()` and `poCmpSpec()`. It'd also be nice if we had more testing for weird string cases in `poCmpSpec`, such as "Bob\x00" and "Bob\x00\x00...(19)".